WuV/onto_general.tex

```diff
@@ -71,6 +71,16 @@ \section{General Principles}\label{sec:onto:principles}
  This part is commonly called the \textbf{ABox} (A for assertional).
 \end{compactitem}
 
+Some sentences in following paragraph partially copied from \url{https://en.wikipedia.org/w/index.php?title=Abox&oldid=892872890}. License: \url{https://en.wikipedia.org/w/index.php?title=Special:CiteThisPage&page=Abox&id=892872890&wpFormIdentifier=titleform}.
+TBox declarations tend to be more permanent within a knowledge base and are used as a schema.
+In contrast, ABox declarations are much more dynamic in nature and tend to represent instance data.
+For instance, we can understand relational databases as ontologies made up of table definitions as TBox declarations and table rows as ABox declarations.
+There, both kind of declarations are even separately stored for efficiency reasons, and in particular, the concrete way of representing table rows
+as bits is determined by the corresponding table definitions.
+However, such a hard distinction is not necessarily the case for other ontology languages.
+For example, in the ontology language OWL or in general in triplestores, both kind of declarations are stored next to each other and the syntactic representation distinction blurs.

```

Comment:
I think triplestores haven't been introduced at this point, but are mentioned here.